damus/Views/PubkeyView.swift

```diff
@@ -72,7 +74,7 @@ struct PubkeyView: View {
                                 .resizable()
                                 .contentShape(Rectangle())
                                 .foregroundColor(colorScheme == .light ? DamusColors.darkGrey : DamusColors.lightGrey)
-                                .frame(width: 20, height: 20)
+                                .frame(width: 15, height: 15)
```

Comment:
These sizing changes affect other instances of `PubkeyView`, such as in the profile view and the profile action sheet. This smaller size makes sense on the side menu, but maybe not in the other views. Was this intended?

If this is not intended, perhaps we can implement an `OptionSet` argument on `PubkeyView` to allow the caller to choose how to display this? (Similar to how we handle different display styles on `EventView` by using `EventViewOptions`)